src/routes/Admin/admin.route.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response, Router } from 'express';
+import { isAdmin } from '../../middleware/isAdmin.middleware';
+import UserModel from '../../models/user.model';
+import { isAuth } from '../../middleware/authenticateToken.middleware';
+
+export const Adminrouter: Router = Router();
+
+Adminrouter.get(
+  '/users',
+  isAuth,
+  isAdmin,
+  async (req: Request, res: Response) => {
+    const users = UserModel.find();
```

Comment:
Also we will be using services files for data ase query so that we can add metrix later if we want